src/wallet/rpc/wallet.cpp

```diff
@@ -701,6 +701,59 @@ RPCHelpMan simulaterawtransaction()
     };
 }
 
+static RPCHelpMan migratewallet()
+{
+    return RPCHelpMan{"migratewallet",
+        "\nEXPERIMENTAL warning: This call may not work as expected and may be changed in future releases\n"
```

Comment:
nit: No need for the `\n`.
```suggestion
        "EXPERIMENTAL warning: This call may not work as expected and may be changed in future releases\n"
```

Reply:
Done